custom_components/weenect/device_tracker.py

```diff
@@ -179,5 +179,14 @@ def extra_state_attributes(self):
                 res["speed"] = self.coordinator.data[self.id]["position"][0]["speed"]
                 res["course"] = self.coordinator.data[self.id]["position"][0]["direction"]
                 res["PDOP"] = self.coordinator.data[self.id]["position"][0]["pdop"]
+                res["last_position_date"] = self.coordinator.data[self.id]["position"][0]["date_tracker"]
+                res["gsm_text"] = self.coordinator.data[self.id]["position"][0]["gsm_text"]
+                res["gsm_color"] = self.coordinator.data[self.id]["position"][0]["gsm_color"]
+                res["accuracy_text"] = self.coordinator.data[self.id]["position"][0]["accuracy_text"]
+                res["accuracy_color"] = self.coordinator.data[self.id]["position"][0]["accuracy_color"]
```

Comment:
What is the added value in providing these in addition to `radius` which is already used to determine the GPS location accuracy?

Reply:
That's just the «official hint». You can determine the GPS location accuracy by yourself (for example, you can determine radius 15, 20, 50 is a good accuracy, according to your case) or you can let Weenect (accuracy_text) give you hint about that.